src/System.Net.Sockets/tests/FunctionalTests/ReceiveMessageFrom.cs

```diff
@@ -41,5 +41,111 @@ public void Success()
                 }
             }
         }
+
+        [OuterLoop] // TODO: Issue #11345
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/987
+        public void Success_IPv6()
+        {
+            if (Socket.OSSupportsIPv6)
```

Comment:
There's some other test code that does the following; seem reasonable?

        [Trait("IPv6", "true")]